ci-operator/config/openshift/release/openshift-release-master__ci-4.9.yaml

```diff
@@ -201,13 +225,27 @@ tests:
     env:
       TEST_SUITE: openshift/conformance/parallel
     workflow: openshift-e2e-openstack-ipi
+- as: e2e-openstack-techpreview-parallel
```

Comment:
Why is parallel in the test name? It's assumed that's what's running unless otherwise specified.

Reply:
I've copied this from the test above, `e2e-openstack-parallel` on line 221 in this diff. Not sure why it's called that but this is just keeping consistency with the rest of the openstack tests